config/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "config",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Check out the `package.json` for some of the other modules.  You should:

1. Make sure this is included in `lerna.json`
2. Get rid of the `test` script that failes
3. Add a `compile` script like the others

Reply:
Hmmm, this should have been in there. Thanks for pointing it out. I'll add it.

Reply:
It is still angry ` 22:17  error  Strings must use singlequote  quotes`

Reply:
Done